control-plane/grpc/build.rs

```diff
@@ -16,6 +16,8 @@ fn main() {
                 "proto/v1/ha/node_agent.proto",
                 "proto/v1/ha/csi_node_nvme.proto",
                 "proto/v1/snapshot/snapshot.proto",
+                "proto/v1/frontend_node/frontend_node.proto",
```

Comment:
would it make sense to have frontend_node.proto inside node/ ?

Reply:
Both are completely different things, not sure if we can bundle them together.

Reply:
hmm they are both node protos, what I mean is like so:
>v1/node/frontend_node.proto
v1/node/target_node.proto

Reply:
Okay, should I bundle the `frontend_node_registration.proto` inside the `frontend_node.proto` otherwise maybe put that under seperate `registration` folder?

Reply:
possibly, or app_node_registration.proto?